app/i18n/de/sub.php

```diff
@@ -34,10 +34,10 @@
 		),
 		'clear_cache' => 'Nicht cachen (für defekte Feeds)',
 		'content_action' => array(
-			'_' => 'Content action when fetching the article content',	// TODO - Translation
-			'append' => 'Add after existing content',	// TODO - Translation
-			'prepend' => 'Add before existing content',	// TODO - Translation
-			'replace' => 'Replace existing content',	// TODO - Translation
+			'_' => 'Behandlung von Feed-Inhalt beim Herunterladen von Artikelinhalt',
+			'append' => 'Artikelinhalt hinter Feed-Inhalt platzieren',
```

Comment:
Not something like einfügen?

Reply:
you are right, it's too early for my brain. platzieren is not the wrong word, but I guess hinzufügen simply fits best in this case.

Reply:
Haha, I actually thought you made a conscious decision against hinzufügen.